week-2-practise/level-2/4-slice.js

```diff
@@ -5,7 +5,7 @@
 // - remove the item at the specified index
 
 function remove(arr, index) {
-  return; // complete this statement
+  return arr.slice(0, index).concat(arr.slice(index + 1));
```

Comment:
It's a very good independently achieved solution, although array.splice() would have been easier